eduNEXT/xblock-filesmanager
# feat: adding new description for instructions and remove features

### Description

Description for the instructions updated and features list for the instructions removed.

BEFORE            |  NOW
:-------------------------:|:-------------------------:
![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/dbf01b5c-59ac-406b-9b64-7dbc9540cc4a)  |  ![](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/5c4345fd-e679-4cf3-b870-40e023dee603)

#### Edit view
![edit-view-after](https://github.com/eduNEXT/xblock-filesmanager/assets/134975835/c9e88552-fbda-4bc7-b61d-4430d45d896a)



### How to test it

- Check this branch`jv/feat-remove-feature-instructions` in the XBlock files manager

1. use node 18
 ```
nvm use
```

2. install dependencies
```
yarn install 
```
3 run to see any change in the react code 
```
yarn dev 

```

**Merge checklist:**
Check off if complete *or* not applicable:
